src/components/chips/TaskTitleWrapper.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { createFragmentContainer } from 'react-relay';
+import { graphql } from 'babel-plugin-relay/macro';
+import Head from 'react-helmet';
+
+// has to be JS because otherwise the property throws a TS error
```

Comment:
Let's make it TS. I think if you'll fix the fragment down below then TS compiler will be happy.